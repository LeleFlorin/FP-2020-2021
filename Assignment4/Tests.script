:l Solutions.hs

strWords "Whats up doc"
strWords "Beware the Jabberwock my son"

piglatinize "Whats up doc"
piglatinize "An apple a day keeps the doctor away"

apprPi

topWords 10 "I know that you know that I know"
topWords 3 "I know that you know that I know"
topWords 6 "An apple a day keeps the doctor away"