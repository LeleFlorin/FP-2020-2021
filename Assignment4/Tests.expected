["Whats","up","doc"]
["Beware","the","Jabberwock","my","son"]

"hats-Way up-hay oc-day"
"An-hay apple-hay a-hay ay-day eeps-kay he-tay octor-day away-hay"

-3.1415926535897927

[("know",3),("I",2),("that",2),("you",1)]
[("know",3),("I",2),("that",2)]
[("An",1),("a",1),("apple",1),("away",1),("day",1),("doctor",1)]